scripts/merge-forward.sh

```diff
@@ -8,6 +8,7 @@ git checkout -B v2-main origin/v2-main
 
 # Some package rules differ between v1 and v2, most notably which packages can be public vs private.
 # These differences are fixable via 'pkglint', so we run that and commit the delta (if any).
-lerna run pkglint && { git diff --quiet || git commit -am 'automatic pkglint fixes'; }
+yarn pkglint
+git diff --quiet || git commit -am 'automatic pkglint fixes'
```

Comment:
This could also just be

```suggestion
git commit -am 'automatic pkglint fixes' || true
```

Reply:
I like it with the `diff` better. I'm not _exactly_ sure what the edge case is, but your suggestion would pass if we need the commit, but it fails for some reason. I'd prefer we first determine we need the commit or not, then have the exit status be based on the success or failure of that commit.